blazor/diagram/ports/interaction.md

```diff
@@ -13,7 +13,8 @@ The port can be used to create connector by enabling the `Draw` in the [PortCons
 
 ## How to draw connector from the port
 
-Diagram provides the support to draw the connector in the port.
+Diagram provides the support to draw the connector in the port. By default, the connector segment type is set to  [Orthogonal](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ConnectorSegmentType.html#Syncfusion_Blazor_Diagram_ConnectorSegmentType_Orthogonal)
```

Comment:
The Diagram provides the support to draw the connector in the port. By default, the connector segment type is set to [Orthogonal](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ConnectorSegmentType.html#Syncfusion_Blazor_Diagram_ConnectorSegmentType_Orthogonal).